estuary/auth.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from flask_oidc import OpenIDConnect
+
+
+class EstuaryOIDC(OpenIDConnect):
+    """Customized version of flask_oidc.OpenIDConnect."""
+
+    def load_secrets(self, app):
+        """
+        Get the configuration required to introspect tokens using flask_oidc.
+
```

Comment:
You might add a comment here explaining *why* you wanted to overwrite this method.

The default `load_secrets` implementation requires a second mega config file... and that seemed like overkill.  This lets us keep all the config in the same, single app config file.